theories/algebra/StdBigop.ec

```diff
@@ -167,6 +172,11 @@ lemma sumr_const (P : 'a -> bool) (x : real) (s : 'a list):
   BRA.big P (fun (i : 'a) => x) s = (count P s)%r * x.
 proof. by rewrite sumr_const RField.intmulr RField.mulrC. qed.
 
+lemma sumr_const_val (P : 'a -> bool) (F : 'a -> real) (x : real) (s : 'a list):
+     (forall (i : 'a), P i => F i = x) 
+  => BRA.big P F s = (count P s)%r * x.
+proof. by rewrite -sumr_const &(BRA.eq_bigr). qed.
```

Comment:
Same as `big_constz_val`

Reply:
Same comment as for `big_constz_val` (except replacing `big_constz` by `sumr_const`).